x/gov/model.go

```diff
@@ -14,19 +14,30 @@ var validTitle = regexp.MustCompile(`^[a-zA-Z0-9 _.-]{4,128}$`).MatchString
 const maxElectors = 2000
 
 func (m Electorate) Validate() error {
-	if !validTitle(m.Title) {
-		return errors.Wrap(errors.ErrInvalidInput, fmt.Sprintf("title: %q", m.Title))
-	}
 	switch n := len(m.Electors); {
 	case n == 0:
 		return errors.Wrap(errors.ErrInvalidInput, "electors must not be empty")
 	case n > maxElectors:
 		return errors.Wrap(errors.ErrInvalidInput, fmt.Sprintf("electors must not exceed: %d", maxElectors))
+	case !validTitle(m.Title):
+		return errors.Wrap(errors.ErrInvalidInput, fmt.Sprintf("title: %q", m.Title))
```

Comment:
Here and a few more places, you can use `errors.Wrapf` instead of `fmt.Sprintf`.

Reply:
🤔 I thought`Wrapf` was deprecated. Good to know.